packages/lib-classifier/src/plugins/tasks/SurveyTask/components/SurveyTask.spec.js

```diff
@@ -1,34 +1,52 @@
 import { shallow } from 'enzyme'
-import { types } from 'mobx-state-tree'
+import { Drop } from 'grommet'
 import React from 'react'
 
-import SurveyTask from './SurveyTask'
 import { default as Task } from '@plugins/tasks/SurveyTask'
+import Choice from './components/Choice'
 import Chooser from './components/Chooser'
+import SurveyTask from './SurveyTask'
 
 describe('SurveyTask', function () {
+  let wrapper
   const task = Task.TaskModel.create({
     taskKey: 'T0',
     type: 'survey'
   })
-  const annotation = task.defaultAnnotation()
 
-  describe('survey task', function () {
-    let wrapper
-    
+  before(function () {
+    wrapper = shallow(
+      <SurveyTask
+        task={task}
+      />
+    )
+  })
+
+  it('should render without crashing', function () {
+    expect(wrapper).to.be.ok()
+  })
+
+  it('should render a Chooser component', function () {
+    expect(wrapper.find(Chooser)).to.have.lengthOf(1)
+  })
+
+  describe.skip('with selectedChoice', function () {
```

Comment:
I'm having difficulty with these related tests. I think it's due to the `choiceTargetRef`. I'm not sure how to define it for testing purposes.